projects/angular/src/layout/nav/API.md

```diff
@@ -0,0 +1,56 @@
+# Nav - Style API
+
+## CSS Variables
+
+| CSS Variable Name              | Description                                                                              |
+| ------------------------------ | ---------------------------------------------------------------------------------------- |
+| --clr-link-active-color        | Changes the text color when the link is active.                                 |
+| --clr-link-color               | Changes the link text color.                                                |
+| --clr-link-hover-color         | Changes the text color when user hovers on element.           |
+| --clr-link-visited-color       | Changes the text color when user clicked the link.                      |
+| --clr-custom-links-hover-color | Changes the neutral color of the link text when user hovers on element.  |
+
+## Class names
+
+| Class name | Description                                                    |
+| ---------- | -------------------------------------------------------------- |
+| clr-link   | The "color" property can be used to change the color of links. |
```

Comment:
This class name doesn't exist either.

Reply:
Hi, @kevinbuhmann removed the class name and implemented it as per your suggestion.